File: scratch/search_admin.py
import os
import re

base_dir = os.path.dirname(os.path.abspath(__file__))
admin_html_path = os.path.join(base_dir, '..', 'admin.html')

with open(admin_html_path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

print("--- BÚSQUEDA EN admin.html (Python) ---")
queries = ["club_teams", "ExtractedTeamsData", "teams_data_auto.js", "teams_data", "SINCRO"]

for query in queries:
    print(f"\nResultados para '{query}':")
    for idx, line in enumerate(lines):
        if re.search(re.escape(query), line, re.IGNORECASE):
            print(f"Línea {idx + 1}: {line.strip()[:150]}")

File: scratch/search_admin.js
const fs = require('fs');
const path = require('path');

const adminHtmlPath = path.join(__dirname, '..', 'admin.html');
const adminHtml = fs.readFileSync(adminHtmlPath, 'utf8');

console.log("--- BÚSQUEDA EN admin.html ---");
const queries = [/club_teams/gi, /ExtractedTeamsData/gi, /teams_data_auto\.js/gi, /teams_data/gi, /SINCRO/gi];
queries.forEach(regex => {
    let match;
    console.log(`\nResultados para ${regex}:`);
    const lines = adminHtml.split('\n');
    lines.forEach((line, index) => {
        if (regex.test(line)) {
            console.log(`Línea ${index + 1}: ${line.trim().substring(0, 150)}`);
        }
    });
});
